internal/discord/handlers.go

```diff
@@ -0,0 +1,251 @@
+package discord
+
+import (
+	"fmt"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+var (
+	// I am too dumb and tired to figure out how to set permission restrictions for the roles without these vars.. sorry..
+	kickPermission    int64 = discordgo.PermissionKickMembers
+	mutePermission    int64 = discordgo.PermissionModerateMembers
+	banPermission     int64 = discordgo.PermissionBanMembers
+	channelPermission int64 = discordgo.PermissionManageChannels
+	messagePermission int64 = discordgo.PermissionManageMessages
+)
+
+func (d *Discord) AddCommands(s *discordgo.Session, event *discordgo.Ready) {
+	fmt.Printf("Initializing Discord...\n")
+
+	for _, discordGuild := range event.Guilds {
+
+		// Function definitions
+		var commands []*discordgo.ApplicationCommand
+		commands = append(commands,
+			KickCommand,
+			MuteCommand,
+			UnmuteCommand,
+			BanCommand,
+			UnbanCommand,
+			RemoveNicknameCommand,
+			SetNicknameCommand,
+			SlowmodeCommand,
+			SlowmodeOffCommand,
+			PurgeCommand,
+		)
+
+		fmt.Printf("Adding commands...\n")
+		commandList, err := s.ApplicationCommandBulkOverwrite(event.User.ID, discordGuild.ID, commands)
+		fmt.Printf("List of successfully created commands:\n")
+		for _, command := range commandList {
+			fmt.Printf("\t%v\n", command.Name)
+		}
+		if err != nil {
+			fmt.Printf("Could not add some commands: %v \n", err)
+		}
+	}
+}
+
+var KickCommand = &discordgo.ApplicationCommand{
+	Name:                     "kick",
+	DefaultMemberPermissions: &kickPermission,
+	Description:              "Kick the specified user and notify the user why via DMs.",
+	Options: []*discordgo.ApplicationCommandOption{
+		{
+			Type:        discordgo.ApplicationCommandOptionUser,
+			Name:        "user",
+			Description: "User being kicked",
+			Required:    true,
+		},
+		{
+			Type:        discordgo.ApplicationCommandOptionString,
+			Name:        "reason",
+			Description: "Reason for kick",
+			Required:    false,
+		},
+	},
+}
+
+var MuteCommand = &discordgo.ApplicationCommand{
+	Name:                     "mute",
+	DefaultMemberPermissions: &mutePermission,
+	Description:              "Mute the specified user.",
+	Options: []*discordgo.ApplicationCommandOption{
+		{
+			Type:        discordgo.ApplicationCommandOptionUser,
+			Name:        "user",
+			Description: "User being muted",
+			Required:    true,
+		},
+		{
+			Type:        discordgo.ApplicationCommandOptionInteger,
+			Name:        "duration",
+			Description: "Duration in minutes",
```

Comment:
We should include hours/days

Reply:
Would you prefer multiple fields for minutes/hours/days or just a single field where the input is like "1m", "1h", "1d"?

Reply:
should be  "1m", "1h", "1d"
not multi fields

Reply:
for clarity:
![image](https://github.com/naurffxiv/moddingway/assets/44913669/c14dccad-8d86-45b1-bff4-0089681f971d)